src/components/tasks/card/index.tsx

```diff
@@ -47,10 +44,19 @@ const Card: FC<Props> = ({
     );
     const { SUCCESS, ERROR } = ToastTypes;
     const isUserAuthorized = useContext(isUserAuthorizedContext);
-    const [taskTagLevel, setTaskTagLevel] = useState<taskItem[]>();
+
     const [showEditButton, setShowEditButton] = useState(false);
     const [keyLongPressed] = useKeyLongPressed();
-    const [isLoading, setIsLoading] = useState<boolean>(false);
+
+    const {
+        data: taskTagLevel,
+        isError,
+        isLoading,
+    } = useGetTaskTagsQuery({
+        itemId: cardDetails.id,
+    });
```

Comment:
should this be `error` instead of `isError`, check the documentation [here](https://redux-toolkit.js.org/rtk-query/overview#configure-the-store)

Reply:
yes it is working @vinayak-trivedi